src/sage/homology/algebraic_topological_model.py

```diff
@@ -100,8 +100,8 @@ def algebraic_topological_model(K, base_ring=None):
 
         sage: from sage.homology.algebraic_topological_model import algebraic_topological_model
         sage: RP2 = simplicial_complexes.RealProjectivePlane()
-        sage: phi, M = algebraic_topological_model(RP2, GF(2))
-        sage: M.homology()
+        sage: phi, M = algebraic_topological_model(RP2, GF(2))                          # optional - sage.rings.finite_rings
+        sage: M.homology()                                                              # optional - sage.rings.finite_rings
         {0: Vector space of dimension 1 over Finite Field of size 2,
```

Comment:
No `sage.modules`?

Reply:
All of `sage.homology` is intended to be shipped by a distribution that provides linear algebra (directly or via dependencies). That's why there are no `# optional - sage.modules` annotations anywhere in `sage.homology`.